src/components/header/header.styles.ts

```diff
@@ -0,0 +1,114 @@
+import styled, {keyframes} from "styled-components";
+
+export const MainContainer = styled.div`
```

Comment:
Algo que no se le da bola, pero que es muy importante es la semántica del HTML. Si bien podemos tratar todo como simples "divs", HTML te proporciona etiquetas específicas para cada cosa, como por ejemplo `<header>`
Te dejo un [link ](https://www.w3schools.com/html/html5_semantic_elements.asp) al respecto.

Estaría bueno que le demos bola :) por lo tanto, esto en vez de un div, debería ser un header


```suggestion
export const MainContainer = styled.header`
```